codes/classical/matrices/rank_metric.yml

```diff
@@ -0,0 +1,38 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: rank_metric
+physical: matrices
+logical: q-ary_digits
+
+name: 'Rank metric code'
+introduced: '\cite{doi:10.1016/0097-3165(78)90015-8}'
+
+
+description: |
+  Encodes \(k\) states in an \(m\times \n)-dimensional \textit{matrix} over \(GF(q)\), using the rank of the difference of matrices as the distance function.
+  Distance \(d\) is the minimum rank matrix in the code.
```

Comment:
Correct line 16 as follows: The distance \(d\) equals the minimum rank of a nonzero matrix in the code.